lib/rewards-bamboohr/templates/anniversary.rb

```diff
@@ -0,0 +1,45 @@
+module RewardsBamboohr
+  module Templates
+    class Anniversary < Base
+      DEFAULT_TEMPLATE = "+100 Happy %{number} Anniversary @%{username}".freeze
+
+      def generate
+        template % { username: username, number: anniversary_number }
+      end
+
+      private
+
+      def anniversary_number
+        ordinalize(year(Date.today) - year(bamboohr_data["hireDate"]))
+      end
+
+      def year(date)
+        return unless date
```

Comment:
Would you like replace this with `date.to_date.year rescue Date.current.year`?

Reply:
Yep, why not